pkg/pillar/types/blob.go

```diff
@@ -47,6 +47,7 @@ type BlobStatus struct {
 	Progress uint
 	// ErrorAndTimeWithSource provide common error handling capabilities
 	ErrorAndTimeWithSource
+	RetryCount int
```

Comment:
just nitpicking
```suggestion
	RetryCount uint32
```
or did I miss that we use a negative `RetryCount` somewhere?